ui/js/graphs.js

```diff
@@ -423,6 +423,19 @@ perf.controller('GraphsCtrl', [
     }
 
     $scope.repoName = $stateParams.projectId;
+    
+    var zoomStringify = function(){
```

Comment:
This only seems to be used in one place, I'd just put the logic here inline where it's used.